scRNA-seq/00-scRNA_introduction.md

```diff
@@ -1,39 +1,33 @@
 
-![**Overall view of full-length and tag-based workflows**](diagrams/overview_workflow.png)
+In *full-length* scRNA-seq, cells are physically separated into individual wells of a plate and are often also sorted by other means (e.g., Fluorescence Activated Cell Sorting).
+With full-length scRNA-seq, each cell is sequenced individually and has its own fastq file. 
+An example of full-length scRNA-seq is Smart-seq2 [(Picelli _et al._ 2014)](https://www.nature.com/articles/nprot.2014.006).
 
-### Full-length scRNA-seq  
-*Example:* Smart-seq2 [(Picelli _et al._ 2014)](https://www.nature.com/articles/nprot.2014.006)    
-Cells are physically separated into individual wells of a plate and are often also sorted by other means (e.g., Fluorescence Activated Cell Sorting).
-Each cell is then sequenced individually and has its own fastq file.
-(This will result in two fastq files if this is paired-end sequencing.)
-The data preprocessing steps for these types of scRNA-seq data are similar to bulk RNA-seq methods.
+For the purposes of this tutorial, we will focus on tag-based scRNA-seq, but it is important to keep in mind that the pre-processing steps and the biases to look out for in post-processing vary based on technology and how the cells are sorted.
 
-#### Pros:  
-- Can be paired-end sequencing which has less risk for 3' bias.  
-- More complete coverage of transcripts, which may be better for transcript discovery purposes.   
+For more extensive background on single-cell experimental methods, Kiselev _et al._ have a very good [tutorial for scRNA-seq](https://scrnaseq-course.cog.sanger.ac.uk/website/index.html).
 
-#### Cons:  
-- Is not very efficient (generally 96 cells per plate).  
-- Takes much longer to run (days/weeks depending on sample size).
-- Much more expensive.  
+![**Overall view of scRNA-seq tag-based workflow**](diagrams/overview_workflow.png)
 
 ### Tag-based scRNA-seq  
-*Example:* 10X Genomics [(Zheng _et al._ 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601)    
-Cells are separated by emulsion/droplets, and individual cells are given barcodes.
-Everything is then sequenced.
-These types of methods, because they are newer, are more likely to have Unique Molecular Identifiers (UMIs) [(Islam _et al._ 2014)](http://www.nature.com/doifinder/10.1038/nmeth.2772). 
-These are additional barcodes added before amplification which allow the identification of PCR duplicates, allowing control for PCR amplification errors and biases.
-Individual samples have two fastq files: one for the cell barcodes and another with the individual reads.
+*Example:* 10X Genomics [(Zheng _et al._ 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601)
+Individual cells are separated by emulsion/droplets prior to cell lysis. 
+Transcripts from each cell are then tagged with two barcodes: a unique cell barcode and a Unique Molecular Identifier (UMI) 
```

Comment:
Changing wording to reduce ironic redundancy of "unique". 

I also added the reference for UMIs here, though I guess that could be confusing given the fact that UMIs are not necessarily specific to this method. But I think it is okay.
 
```suggestion
Transcripts from each cell are then tagged with two barcodes: a cell-specific barcode and a Unique Molecular Identifier (UMI) [(Islam _et al._ 2014)](http://www.nature.com/doifinder/10.1038/nmeth.2772). 
```